Tests/Common/Util/SdkClientTest.swift

```diff
@@ -0,0 +1,87 @@
+@testable import CioInternalCommon
+import Foundation
+import SharedTests
+import XCTest
+
+class SdkClientTest: UnitTest {
+    private let deviceInfoMock = DeviceInfoMock()
+    private let defaultClientString = "iOS Client/\(SdkVersion.version)"
```

Comment:
this should be defaulting to `SdkClient.version`

Reply:
No. This is for testing default value and should be `SdkVersion.version`